pkgs/top-level/default.nix

```diff
@@ -80,7 +83,7 @@ in let
   boot = import ../stdenv/booter.nix { inherit lib allPackages; };
```

Comment:
If we do want all bootstraping stages to include the overlay unconditionally, we could instead "partially apply" overlays to all packages (3 lines above the `inherit lib nixpkgsFun;`, but I cannot comment there because unchanged).